install.sh

```diff
@@ -750,7 +750,7 @@ case "$SHELL" in
 esac
 if [[ "$UNAME_MACHINE" == "arm64" ]] || [[ -n "${HOMEBREW_ON_LINUX-}" ]]; then
   cat <<EOS
-- Add Homebrew to your ${tty_bold}PATH${tty_reset} in ${tty_underline}${shell_profile}${tty_reset}:
+- Run these two commands to add Homebrew to your ${tty_bold}PATH${tty_reset} via ${tty_underline}${shell_profile}${tty_reset}:
```

Comment:
Actually, let's make this super explicit:
```suggestion
- Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
```